src/graph.py

```diff
@@ -68,3 +70,66 @@ def adjacent(self, key, val):
             return val in self.g[key]
         except KeyError:
             return "Node does not exist"
+
+    def depth_first_traversal(self, start):
+        """Traverse the graph by depth."""
+        stack = [start]
```

Comment:
Why not use the Stack() object that you created, instead of assigning a list to "stack"?